Podfile.lock

```diff
@@ -106,7 +106,7 @@ PODS:
   - Nimble (9.2.0)
   - PromisesObjC (1.2.12)
   - Quick (3.1.2)
-  - TwilioVideo (4.5.0-rc1)
+  - TwilioVideo (4.5.0-rc4)
```

Comment:
Needs fixing before merge. I am using the https remote and this branch is setup to use the git remote.

Reply:
Cool shouldn't hold up this PR, just the feature branch.